build-system/tasks/process-github-issues.js

```diff
@@ -119,67 +124,118 @@ function updateGitHubIssues() {
           let milestoneTitle;
           let milestoneState;
           let hasPriority = false;
+          let hasCategory = false;
           let issueNewMilestone = MILESTONE_PENDING_TRIAGE;
-
-          // Get the title and state of the milestone
+          const assignee = issue['assignee'];
+          let assigneeName = '';
+          const issueLastUpdate = issue.updated_at;
+          let biweeklyUpdate = true;
+          let quartelyUpdate = true;
+          if (getLastUpdate(issueLastUpdate) > 89) {
+            quartelyUpdate = false;
+            biweeklyUpdate = false;
+          }
+          else if (getLastUpdate(issueLastUpdate) > 14) {
```

Comment:
Same comment as above. Use `const`s at the top of this file instead of hardcoding numbers like `14`.

Reply:
done